README.md

```diff
@@ -39,13 +39,43 @@ jobs:
           echo "Your build commands go here"
 
       - name: Run the CoGuard CLI Action
-        uses: coguardio/coguard-scan-action@v0.2.2
+        uses: coguardio/coguard-scan-action@v0.2.37
         with:
           dockerImageName: YourImageName
           username: ${{ secrets.CoGuardUserName }}
           password: ${{ secrets.CoGuardPassword }}
 ```
 
+## Scanning and upload as SARIF
+
+```yaml
+name: Including CoGuard into your GitHub Action and upload as SARIF
+on: [push]
+jobs:
+  create-image-and-run-coguard:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Build your docker image
+        run: |
+          echo "Your build commands go here"
+
+      - name: Run the CoGuard CLI Action
+        uses: coguardio/coguard-scan-action@v0.2.37
+        with:
+          dockerImageName: YourImageName
+          username: ${{ secrets.CoGuardUserName }}
+          password: ${{ secrets.CoGuardPassword }}
+          outputFormat: sarif
+      - name: Upload SARIF file
+        uses: github/codeql-action/upload-sarif@a57c67b89589d2d13d5ac85a9fc4679c7539f94c
+        with:
+          sarif_file: result.sarif.json
+          category: CoGuard
+      - name: Fail job if there is a result
```

Comment:
I don't like that part... maybe we should have it directly here as output-and-upload?